packages/editor/src/lib/editor/managers/TextManager.ts

```diff
@@ -146,6 +147,7 @@ export class TextManager {
 				// the character itself. we're only interested in the character.
 				const rects = range.getClientRects()
 				const rect = rects[rects.length - 1]!
+				if (!rect) whyAmIRunning()
```

Comment:
Should this throw an error?

Reply:
this looks like maybe leftover from testing, but could be wrong